playbooks/templates/standalone_parameters.yaml.j2

```
@@ -58,7 +58,9 @@ parameter_defaults:
   NeutronFlatNetworks: {{ neutron_flat_networks }}
 {% if sriov_interface is defined %}
   OVNCMSOptions: "enable-chassis-as-gw"
+{% if neutron_physical_dev_mappings is not defined %}
   NeutronPhysicalDevMappings: "hostonly-sriov:{{ sriov_interface }}"
+{% endif %}
```

Wasn't the intention to have `neutron_physical_dev_mappings` overwrite the setting?

In which case, perhaps something like:
```suggestion
{% if neutron_physical_dev_mappings is defined %}
  NeutronPhysicalDevMappings: {{ neutron_physical_dev_mappings }}
{% else %}
  NeutronPhysicalDevMappings: "hostonly-sriov:{{ sriov_interface }}"
{% endif %}
```

---

you're very right, thanks for this review.